gui-wm/hyprland/hyprland-0.27.2-r1.ebuild

```diff
@@ -75,7 +75,7 @@ pkg_setup() {
 src_prepare() {
 	if use video_cards_nvidia; then
 		cd "${S}/subprojects/wlroots" || die
-		eapply "${S}/nix/wlroots-nvidia.patch"
+		eapply "${FILESDIR}/nvidia-0.27.2.patch"
```

Comment:
https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html

Id go with hyprland-0.27.2-wlroots-nvidia.patch personally.

Reply:
ok i will do that changes